lib/assessment.js

```diff
@@ -4,31 +4,62 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
-
+var min = function min(nums) {
+  //grab the first number to compare against
+  var minValue = nums[0];
+  // loop through the remaining numbers
+  for(var i = 1; i < nums.length; i++){
+    if(nums[i] < minValue){
+      //replace minValue with new number
+      minValue = nums[i];
+    }
+    return minValue;
+  }
```

Comment:
If you replace all instances of `nums` with `arguments` and move the `return` statement out of the loop body, this will be a correct solution. 1/2